docs/reference/bpmn-processes/message-events/message-events.md

```diff
@@ -49,6 +47,31 @@ The `correlationKey` is an expression that usually [accesses a variable](/compon
 
 To correlate a message to the message event, the message is published with the defined name (e.g. `Money collected`) and the **value** of the `correlationKey` expression. For example, if the process instance has a variable `orderId` with value `"order-123"`, the message must be published with the correlation key `"order-123"`.
 
+## Message throw events
```

Comment:
🔧  We should move this paragraph one paragraph up (above Messages), so it is grouped with the other events